xml/src/main/resources/ehcache-core.xsd

```diff
@@ -214,33 +214,22 @@
       <xs:element name="writebehind" minOccurs="0" maxOccurs="1">
         <xs:complexType>
           <xs:sequence>
-           <xs:element name="writedelay" minOccurs="0" maxOccurs="1">
-            <xs:complexType>
-              <xs:simpleContent>
-                <xs:extension base="xs:string">
-                  <xs:attribute type="xs:nonNegativeInteger" name="min" default="0" use="optional"/>
-                  <xs:attribute type="xs:positiveInteger" name="max" default="1" use="optional"/>
-                </xs:extension>
-              </xs:simpleContent>
-            </xs:complexType>
-          </xs:element>
-          <xs:element type="xs:nonNegativeInteger" name="ratelimitpersecond" default="0" minOccurs="0" maxOccurs="1"/>
-          <xs:element type="xs:positiveInteger" name="batchsize" default="1" minOccurs="0" maxOccurs="1"/>
-          <xs:element name="retryAttempts" default="0" minOccurs="0" maxOccurs="1">
-            <xs:complexType>
-              <xs:simpleContent>
-                <xs:extension base="xs:nonNegativeInteger">
-                  <xs:attribute type="xs:positiveInteger" name="delay" default="1" use="optional"/>
-                </xs:extension>
-              </xs:simpleContent>
-            </xs:complexType>
-          </xs:element>
-        </xs:sequence>
-        <xs:attribute type="xs:positiveInteger" name="concurrency" default="1" use="optional"/>
-        <xs:attribute type="xs:nonNegativeInteger" name="size" default="0" use="optional"/>
-        <xs:attribute type="xs:boolean" name="coalesce" use="optional"/>
-      </xs:complexType>
-    </xs:element>
+            <xs:element name="retry" minOccurs="0" maxOccurs="1">
```

Comment:
Kept retry as an element to indicate that both attributes must be provided together.